tests/queries/0_stateless/02227_union_match_by_name.sql

```diff
@@ -0,0 +1,5 @@
+-- Tags: no-backward-compatibility-check:22.5.1.2079
```

Comment:
I don't get why it's not backward compatible. This patch should fix LOGICAL_ERROR, but in the issue we got null pointer dereference.

Reply:
Not only logical error. The order of columns was not correct, and in case of aggregator it may get null pointer dereference.